go/worker/compute/executor/committee/node.go

```diff
@@ -700,71 +687,39 @@ func (n *Node) proposeBatchLocked(batch *protocol.ComputedBatch) {
 		return
 	}
 
-	// Publish commitment to merge committee.
-	spanPublish := opentracing.StartSpan("PublishExecuteFinished(commitment)",
```

Comment:
ugh goodbye tracing

Reply:
Just for the merge part which is obviously gone now. The other tracing should still be there.

Reply:
we still submit the result, but we no longer trace it

Reply:
We can't trace submissions to the consensus layer, but I guess we could leave a span open until the consensus layer confirms. Will change it.

Reply:
Hm, we didn't trace merge submissions to consensus before though.

Reply:
oh ok let's continue not to. we can add it later if we want to see